spec/factories/sites.rb

```diff
@@ -24,7 +24,7 @@
     address3 { Faker::Address.city }
     address4 { Faker::Address.state }
     postcode { Faker::Address.postcode }
-    region_code { 'london' }
+    region_code { 1 }
```

Comment:
why do we need this change? `london` is more descriptive than `1`.

Reply:
Nothing seems to break if I put this change back, so I'm gonna amend the commit that introduced this change.